autonomy/chain/constants.py

```diff
@@ -160,6 +176,10 @@
         "gnosis_safe_same_address_multisig": cast(
             str, os.environ.get("CUSTOM_GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS")
         ),
+        "safe_multisig_with_recovery_module": cast(
+            str, os.environ.get("CUSTOM_SAFE_MULTISIG_WITH_RECOVERY_MODULE_ADDRESS")
+        ),
+        "recovery_module": cast(str, os.environ.get("CUSTOM_RECOVERY_MODULE_ADDRESS")),
```

Comment:
Alternative:
```suggestion
        "safe_multisig_with_recovery_module": cast(
            str, os.environ.get("CUSTOM_GNOSIS_SAFE_PROXY_FACTORY_ADDRESS")
        ),
        "recovery_module": cast(str, os.environ.get("CUSTOM_GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS")),
```

The reasoning is that the CLI user always has to set these variables,,, but they need to be in accordance with the flag `--use-multisig-with-recovery-module`...